main.py

```diff
@@ -72,6 +72,11 @@
 originalVideoFile = os.path.abspath(batchConfig['SETTINGS']['original_video_file_path'].strip("\""))
 srtFile = os.path.abspath(batchConfig['SETTINGS']['srt_file_path'].strip("\""))
 
+# Create the output folder based on the original video file name
+fileName = os.path.basename(originalVideoFile).split(".")[0]
```

Comment:
Isn't this the same as the code in `audio_builder.py`? 

Also, there's already some code to handle the creation of output folders in [Line 479 of the main file](https://github.com/ThioJoe/Auto-Synced-Translated-Dubs/blob/ec20076fda4125ed80ef0b8822f881b4c0d8590b/main.py#LL479-L487), in my opinion, moving this implementation there (or replacing the existing one) would be a better fit for this code.